setup.py

```diff
@@ -56,5 +56,6 @@
             ],
         },
       zip_safe=True,
-      test_suite="Tests",
+      python_requires='>=3',
```

Comment:
add package requirements here as well, otherwise `pip install` will not get the packages (see comment below on pandas)

Reply:
I was following https://packaging.python.org/discussions/install-requires-vs-requirements/

Since testing isn't required for installation, I thought pandas should be left to requirements.txt.